src/components/layout/Grid/Grid.tsx

```diff
@@ -88,29 +77,24 @@ const Grid: FC<GridProps> = ({
     return null;
   }
   return (
-    <GridWrapper
+    <GridParent
+      $align={align}
+      $cols={cols}
+      $gap={gap}
       $overflow={wrapperOverflow}
```

Comment:
Let's drop `wrapperOverflow` since we don't have wrapper anymore. There are 2 cases in platform, both setting this to `visible` so it can be safely removed

Reply:
I removed wrapperOverflow